dashboard/modules/actor/actor_head.py

```diff
@@ -56,20 +56,17 @@ def actor_table_data_to_dict(message):
         "state",
         "name",
         "numRestarts",
-        "taskSpec",
+        "functionDescriptor",
         "timestamp",
         "numExecutedTasks",
     }
     light_message = {k: v for (k, v) in orig_message.items() if k in fields}
-    if "taskSpec" in light_message:
-        actor_class = actor_classname_from_task_spec(light_message["taskSpec"])
+    logger.info(light_message)
+    if "functionDescriptor" in light_message:
+        actor_class = actor_classname_from_func_descriptor(
+            light_message["functionDescriptor"]
+        )
         light_message["actorClass"] = actor_class
-        if "functionDescriptor" in light_message["taskSpec"]:
-            light_message["taskSpec"] = {
-                "functionDescriptor": light_message["taskSpec"]["functionDescriptor"]
-            }
```

Comment:
this looks like we're changing the returned schema -- we used to return `light_message["taskSpec"]["functionDescriptor"]` but now it's just `light_message["functionDescriptor"]`. We likely need to change the dashboard to account for this?

Reply:
The dashboard uses `function descriptor` just for showing, we extract it from the returned schema, no matter in what position it is.